hc/api/management/commands/sendreports.py

```diff
@@ -29,7 +29,8 @@ def add_arguments(self, parser):
 
     def handle_one_run(self):
         now = timezone.now()
-        month_before = now - timedelta(days=30)
+        # time_passed = now - profile.next_report_date
+        month_before = now - timedelta(days=1)
```

Comment:
should the time delta be in single day rather than days, since it's a single day